distribution/docs/src/main/resources/content/_managing/_configuring/reusing-configurations.adoc

```diff
@@ -76,5 +76,12 @@ The same step-by-step process above can be followed when migrating configuration
 ** Keystores
 ** Truststores
 ** Service wrapper `*.conf` files, if the ${branding} is installed as a service
-** Content Directory Monitor Configuration (including URL Resource Reader)
+** Select AdminUI Configurations, Including:
+*** Content Directory Monitor Configuration (including URL Resource Reader)
+*** Web Context Policy Manager
+*** Security STS Guest Claims Handler
+*** Security STS Server
+*** Session
+[WARNING]
+If a supported configuration is being imported across versions, any corresponding `.config` files in the etc directory will not be put into the etc directory of the importing system.
```

Comment:
❓ How does this render with the warning message inside the bullet point list?

Reply:
![image](https://user-images.githubusercontent.com/31221295/64727930-d1b11a00-d48e-11e9-8779-828972e7f87b.png)

Would it be better to put it under that last bullet point, or is this good as is? (after putting URL Resource Reader on its own line)

Reply:
I think that's fine 👍